products/logs/frontend/logsLogic.tsx

```diff
@@ -250,8 +264,7 @@ export const logsLogic = kea<logsLogicType>([
 
                     const response = await api.logs.query({
                         query: {
-                            limit: 100,
-                            offset: values.logs.length,
+                            limit: values.logsLimit,
```

Comment:
This feels... not good. We're just going to hit the same endpoint with more and more entries??

I think you might want to use workflow logs as a reference point and implement timestamp based loading which will be way more effective https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/hog-functions/logs/logsViewerLogic.tsx